appveyor.yml

```diff
@@ -1,8 +1,10 @@
 # This is necessary to use WMF5
-os: unstable  
-install: 
+os: unstable
+install:
     - cinst -y pester
     - git clone https://github.com/PowerShell/DscResource.Tests
+    - ps: Get-PackageProvider -name nuget -ForceBootStrap -Force
+    - ps: nuget.exe install xDscResourceDesigner -source 'https://www.powershellgallery.com/api/v2' -outputDirectory "$env:USERPROFILE\Documents\WindowsPowerShell\Modules\" -ExcludeVersion
```

Comment:
Can you add a comment to the issue on why you are using this instead of Powershell get?